platform/jvm/capture/build.gradle.kts

```diff
@@ -57,12 +60,14 @@ android {
         targetCompatibility = JavaVersion.VERSION_1_8
     }
 
-    // TODO(murki): consider updating to using kotlin.compilerOptions {} block
```

Comment:
I took the opportunity to migrate these options as recommended https://kotlinlang.org/docs/gradle-compiler-options.html#migrate-from-kotlinoptions-to-compileroptions